inc/admin/diagnostics/namespace.php

```diff
@@ -166,6 +166,14 @@ function render_page() {
 	?>
 		<textarea style="width: 800px; max-width: 100%; height: 600px; background: #fff; font-family: monospace;" readonly="readonly" onclick="this.focus(); this.select()"
 				title="<?php _e( 'To copy the system info, click below then press Ctrl + C (PC) or Cmd + C (Mac).', 'pressbooks' ); ?>"><?php echo $output; ?></textarea>
+		<h2><?php _e( 'View Source', 'pressbooks' ); ?></h2>
+		<p>
+		<?php
+		$home_url = home_url();
+		$source_url = ( is_super_admin() ) ? $home_url . '/format/xhtml?debug=prince' : $home_url . '/format/xhtml';
```

Comment:
The permissions to see the diagnostics page is `manage_options`
Source: \Pressbooks\Admin\Diagnostics\add_menu, line ~22

The permissions code for /format/xhtml is `current_user_can( 'edit_posts' )`
Source: \Pressbooks\Modules\Export\Xhtml\Xhtml11::transform, line ~222

The permissions code for /format/xhtml?debug=prince is `if ( is_super_admin() || WP_DEBUG ) {`
Source: \Pressbooks\Modules\Export\Xhtml\Xhtml11::transform, line ~263

I think the 2nd is different than the first because A) we didn't want people using this feature B) We used to just do something silly like `?debug={$_GET['debug'}` but we don't anymore, or ... ?

If this feature is now documented, Maybe it's time we make both of them `current_user_can( 'edit_posts' )`?

The same changes should be made in \Pressbooks\Modules\Export\HTMLBook\HTMLBook